libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -164,8 +164,12 @@
       @apply mr-3 #{!important};
     }
 
+    .o-btn__wrapper {
```

Comment:
`elementsWrapperClass`

Reply:
Sorry, could you expand more? This classname was defined from oruga.

Reply:
I think you can do something like:
```vue
:elements-wrapper-class="{ 'w-auto': loadingWithLabel }"
```
or
```vue
loading-class="w-auto"
```

Reply:
Neither of these work on my side. As there are lots of style changes applied to the `'loading-with-label'`, we should not set separate props only for the changes of `width`. What do you think?